internal/statemachine/classic_states.go

```diff
@@ -275,6 +277,49 @@ func (stateMachine *StateMachine) extractRootfsTar() error {
 	return nil
 }
 
+// germinate runs the germinate binary and parses the output to create
+// a list of packages from the seed section of the image definition
+func (stateMachine *StateMachine) germinate() error {
+	var classicStateMachine *ClassicStateMachine
+	classicStateMachine = stateMachine.parent.(*ClassicStateMachine)
+
+	// create a scratch directory to run germinate in
+	germinateDir := filepath.Join(classicStateMachine.stateMachineFlags.WorkDir, "germinate")
+	err := osMkdir(germinateDir, 0755)
+	if err != nil && !os.IsExist(err) {
+		return fmt.Errorf("Error creating germinate directory: \"%s\"", err.Error())
+	}
+
+	germinateCmd := generateGerminateCmd(classicStateMachine.ImageDef)
```

Comment:
Great question. I hope so, as I'd love to eventually make ubuntu-image a strictly confined snap. However, it's not strictly needed right now as the classic images are generally built with the deb and snap images are built with the snap. I do hope to change that at some point though. That's how it is currently done because there are some complications with the snap bundling livecd-rootfs and running livecd-rootfs commands. So now that we're moving away from that we have the chance to do this right.

Reply:
sounds good (and also mostly a problem for the future)